testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -65,29 +65,41 @@ def _single_frame(self):
 def u():
     return mda.Universe(PSF, DCD)
 
+@pytest.fixture()
+def n_frames(u):
+    return len(u.trajectory)
+
 
-def test_default(u):
+def test_default(u, n_frames):
     an = FrameAnalysis(u.trajectory).run()
-    assert an.n_frames == len(u.trajectory)
-    assert_equal(an.frames, list(range(len(u.trajectory))))
+    assert an.n_frames == n_frames
+    assert_equal(an.found_frames, np.arange(n_frames))
+    assert_equal(an.frames, np.arange(n_frames))
+    times = (an.frames+1)/u.trajectory.dt  # DCD starts at 1
+    assert_almost_equal(an.times, times, decimal=4)
```

Comment:
Probably too pedantic, please ignore if that's the case. I'm thinking it might be good to also test that times returns as expected when using start & stop too. Also in order to make test failure easier to understand, maybe it'd be better to move times-checking to a separate function and use parametrize?

Reply:
> I'm thinking it might be good to also test that times returns as expected when using start & stop too. 

Good point, I've done that, but

> Also in order to make test failure easier to understand, maybe it'd be better to move times-checking to a separate function and use parametrize?

I'm not sure what you had in mind here. I did add an error message for help in understanding.

Reply:
I guess it's more of a personal preference of mine, I like to put separate out tests so as to avoid a series of asserts that test different things, and then you can just have a more descriptive function signature, but honestly it doesn't matter. These changes are just as, if not more, descriptive.